extension/src/contentscript/player.ts

```diff
@@ -20,7 +20,21 @@ export interface Event {
         | "seeked"
         | "pause"
         | "waiting";
-    type: MediaEventType;
+    // Type is undefined when it should not trigger state update
+    type: WatcherState | undefined;
+}
+
+export enum HtmlVideoPlayerEventType {
+    ratechange = "ratechange",
+    seeking = "seeking",
+    progress = "progress",
+    canplaythrough = "canplaythrough",
+    play = "play",
+    playing = "playing",
+    canplay = "canplay",
+    seeked = "seeked",
```

Comment:
```suggestion
    sought = "sought",
```

Reply:
I'm being a grammar nazi here, it's fine if we leave "seeked".

Reply:
Aaaah I finally understand why I didn't feel good about this word.

Unfortunately the whole interned made the mistake https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/seeked_event

Reply:
Ahah oops, it's probably fine then